docs/asus-aura-led-guide.md

```diff
@@ -0,0 +1,136 @@
+# Asus Aura LED (USB-based) controllers
+_Driver API and source code available in [`liquidctl.driver.aura_led`](../liquidctl/driver/aura_led.py)._
+
+__NOTE:__
+Some features of this driver are still being developed and/or refined. Your use of this driver indicates that you acknowledge it is a beta release.
```

Comment:
nit: This can be simply left as:

```
__NOTE:__
This is an is an unstable API and may be changed in the future.
```

Reply:
I've replaced the *NOTE* with this:

> This driver is marked `experimental` because of the need for additional testing and feedback by the community. The API is unstable and may be changed in the future.